src/main/java/crazypants/enderio/machine/farm/FarmersRegistry.java

```diff
@@ -191,6 +192,16 @@ private static void addExtraUtilities() {
     }
   }
 
+  private static void addFlowers() {
+    FarmersCommune.joinCommune(new FlowerPicker().add(
+        GameRegistry.findBlock("minecraft", "yellow_flower"), 
+        GameRegistry.findBlock("minecraft", "red_flower"), 
```

Comment:
Number of reasons:

(a) because I can
(b) because I don't want bug reports about "eio doesn't harvest flowers but it's in the changelog"
(c) because there's a change they get spawned by using bonemeal if the user has a block of grass where there should be a plant. (Tall grass already gets replaced, so that's no issue.)